src/transactions/feed/TransferFeedItem.tsx

```diff
@@ -51,6 +50,7 @@ function TransferFeedItem({ transfer }: Props) {
           recipient={recipient}
           status={transfer.status}
           transactionType={transfer.__typename}
+          transactionSubtype={transfer.type}
```

Comment:
i wonder if we need this, because `__typename` is already different for `TokenTransver` and `JumpstartDeposit`